ansible_collections/arista/avd/galaxy.yml

```diff
@@ -36,9 +36,9 @@ tags: ['arista', 'network', 'networking', 'eos', 'avd', 'cloudvision', 'cvp']
 # L(specifiers,https://python-semanticversion.readthedocs.io/en/latest/#requirement-specification). Multiple version
 # range specifiers can be set and are separated by ','
 dependencies:
-    "arista.cvp": ">=3.10.1"
-    "arista.eos": ">=7.0.0"
-    "ansible.utils": ">=3.0.0"
+    "arista.cvp": ">=3.11.0"
+    "arista.eos": ">=10.0.1"
+    "ansible.utils": ">=5.1.2"
```

Comment:
We need to scan these for python and ansible requirements. So we ensure we are in sync on all our reqs.

Reply:
For ansible.utils we need to bump the python requirement to `netaddr>=0.10.1`